.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+jobs:
+  code:
+    name: Versões NodeJS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 14.x
+          - 15.x
+
+    steps:
+      - name: Obtendo versão mais nova do Codigo
+        uses: actions/checkout@v2
+
+      - name: Configurando NodeJS ${{ matrix.node-version }}
+        uses: actions/setup-node@main
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Instalar as dependências do npm
+        run: npm install
```

```suggestion
        run: npm ci
```

O `npm ci` é o comando recomendado para executar instalação em pipeline. Realiza instalação limpa e garante que o `package-lock.json` está atualizado.